.gitattributes

```diff
@@ -3,3 +3,5 @@
 
 *.cpp diff=cpp
 *.h diff=cpp
+
+*.gltf -diff
```

Comment:
alternatively
```suggestion
*.gltf binary
```

Reply:
ah yeah, that does a few more things than just `-diff`